docs/specs/schema.yml

```diff
@@ -398,19 +398,14 @@ outputs:
   c.mta.json:
   c.raw.page.json:
 ---
-# metadata need to be transformed for SDP
+# Transform metadata in SDP yaml
 inputs:
-  docfx.yml: |
-    fileMetadata:
-      href:
-        c.yml: b.md
+  docfx.yml:
```

Comment:
Supporting schema based transformation for _fileMetadata_ and _globalMetadata_ is a bit cumbersome. The original scenario only applies to metadata defined in the YAML file. #4951